### ISSUE TITLE
[Accessibility] FoodOrder: 'How would you like your steak prepared?' option under Steak is not accessible through the swipe gesture. 

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6291]
### Target Platforms

iOS

### SDK Version

App Version: Version 1.0 (2.3.1-beta.20210603.1)

### Application Name

Visualizer

### Problem Description

[33764423](https://microsoft.visualstudio.com/OS/_workitems/edit/33764423)

'How would you like your steak prepared?' option and its radio button options under Steak are not accessible through the swipe gesture.

### Card JSON

[FoodOrder.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.0/Scenarios/FoodOrder.json)

**Comment by jwoo-msft** [comment]
duplicate of #6417
